import-scripts/dddmmm

```diff
@@ -0,0 +1,404 @@
+#!/bin/bash
```

Comment:
@sheridancbio what is this file name?? lol

Reply:
this was not supposed to be part of the changeset ... it was a temporary file. I'll remove it.